cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -670,6 +597,7 @@ struct ConvertFunctor {
                                                       parse_options_view const& opts,
                                                       bool as_hex)
   {
+    // TODO what's invalid input
```

Comment:
Can you please make these TODOs clearer if we're planning to merge them?
I looked into the code and don't see that invalid input is handled in a robust way in `parse_decimal` and `to_timestamp`. Is that what this TODO needs to remind us of?

Reply:
Yes. this `TODO` is a reminder for future and also to remind reviewers to discuss and decide what is invalid input for this type.